lib/hammer_cli/options/option_family.rb

```diff
@@ -11,7 +11,8 @@ def initialize(options = {})
         @all = []
         @children = []
         @options = options
-        @creator = options[:creator] || Class.new(HammerCLI::Apipie::Command)
```

Comment:
FYI: I'm currently working on a little bit different approach for option creation and this anonymous class will disappear :) So, :+1: for the solution, but it should wait a bit before we get rid of this class.